libs.osgi/build.xml

```diff
@@ -22,4 +22,29 @@
 <project basedir="." default="netbeans" name="libs.osgi">
     <description>Builds, tests, and runs the project org.netbeans.libs.osgi</description>
     <import file="../nbbuild/templates/projectized.xml"/>
+    
+    <!-- A task to check if the binary 'external/resolver-1.2.jar exists -->
```

Comment:
Idea: This should read "A task to check if the binary 'external/osgi.cmpn-4.2.jar exists"